keepalived-vip/main.go

```diff
@@ -85,8 +84,6 @@ func main() {
 		glog.Fatalf("Terminating execution: %v", err)
 	}
 
-	proc.StartReaper()
```

Comment:
Ah, while we're doing this I noticed the public methods in keepalived.go don't have godocs, could you please populate them (and comment above Reload what you just said)